src/ConnectedMode/Binding/SolutionBindingOperation.cs

```diff
@@ -27,8 +27,10 @@
 
 namespace SonarLint.VisualStudio.ConnectedMode.Binding
 {
+    // TODO - rename. The current name is misleading. IBindingConfigWriter?
```

Comment:
This class now does very little (although if you look at the history it used to do an awful lot more e.g. [here](https://github.com/SonarSource/sonarlint-visualstudio/blob/19c6a0fc4e307d9226c5f8bca9b4ebcb6ff5f045/src/Integration/Binding/SolutionBindingOperation.cs)).

The name isn't helpful, but changing it in this PR would make the PR harder to review.